src/metabase/driver/druid.clj

```diff
@@ -74,24 +77,26 @@
                 :type/Text)})
 
 (defn- describe-table [database table]
-  (let [details                      (:details database)
-        {:keys [dimensions metrics]} (GET (details->url details "/druid/v2/datasources/" (:name table) "?interval=1900-01-01/2100-01-01"))]
-    {:schema nil
-     :name   (:name table)
-     :fields (set (concat
-                    ;; every Druid table is an event stream w/ a timestamp field
-                    [{:name       "timestamp"
-                      :base-type  :type/DateTime
-                      :pk?        true}]
-                    (map (partial describe-table-field :dimension) dimensions)
-                    (map (partial describe-table-field :metric) metrics)))}))
+  (ssh/with-ssh-tunnel (:details database)
```

Comment:
It started life as a macro, and i went back and forth on it. If there are other oppinions here, sure let's macro!